### ISSUE TITLE
Running without a region doesn't work

### ISSUE THREAD

**Issue by ernfrid** [genome/bam-readcount#30]
You get lots of errors along the lines of: `Couldn't find the generated tag.` because reads are not pre-processed.

**Comment by fwhelan** [comment]
+1ed

Is there a workaround for this? I can't find an example of the format of [region]?

```bam-readcount -f ../NC_015311.fna bin6.bwa.sort.bam > bin6.bwa.sort.bam.readcount```
Runs but provides the above error; but
```bam-readcount -f ../NC_015311.fna bin6.bwa.sort.bam 1-100 > bin6.bwa.sort.bam.readcount```
Gives the following error:
```Minimum mapping quality is set to 0
[bam_index_load] fail to load BAM index.
BAM indexing file is not available.```

Am I missing something?
Thanks!!

**Comment by ernfrid** [comment]
Yes, using a region should be a work around. The region format should be identical to a samtools region (chrom:start-stop). To do a whole chromosome, use the name (e.g. `bam-readcount -f ../NC_015311.fna bin6.bwa.sort.bam chromosome_name`) . As the error indicates, your BAM must be indexed (using `samtools index`) before you run the command.

**Comment by fwhelan** [comment]
Perfect, got it working! Cheers!

**Comment by elbakrifz** [comment]
I was running the following CMD :  /Users/mac/INSTALL_DIR/HMMcopy/bin/readCounter -w 100000 /Users/mac/Desktop/NGS-/ERR439247-pe.bam > /Users/mac/Desktop/NGS-/ERR439247-pe.wig
and I get : 
BamRandomAccessController WARNING: could not find index file for BAM: /Users/mac/Desktop/NGS-/ERR439247-pe.bam
how should I fix it ?
